lib/restrictions.js

```diff
@@ -53,10 +59,30 @@ export async function insert({restriction} = {}) {
   const collection = database.collections['resource-restriction-restriction'];
   const now = Date.now();
   const meta = {created: now, updated: now};
+  if(!restriction.id) {
+    throw new BedrockError(
+      `Id is required.`,
```

Comment:
```suggestion
      `"restriction.id" is required.`,
```

Reply:
We usually just throw a TypeError for this -- so let's do that here as well.

Reply:
Updated